packages/ckeditor5-alignment/src/alignmentediting.js

```diff
@@ -63,17 +60,13 @@ export default class AlignmentEditing extends Plugin {
 
 		if ( shouldUseClasses ) {
 			// Downcast to only to classes.
```

Comment:
This comment is not reflecting what the code does - it's converting both ways, not only downcasting.